OKR/GO_CORE.md

```diff
@@ -2,6 +2,29 @@
 
 We frame our ongoing work using a process based on quarterly Objectives and Key Results (OKRs). Objectives reflect outcomes that are challenging, but realistic. Results are tangible and measurable.
 
+## 2019 Q1
+
+### Data transfer is fast and efficient
+* It takes less than 48 hours to transfer 1TB dataset over Fast Ethernet (100Mbps)
+* Bitswap refactor + improvements reduce number of duplicate blocks downloaded by 75%
+ 
+### Adding and providing large data sets works efficiently
+* Create Provider Strategies system that is solely responsible for providing
+* Implement provider strategy such that a user can add (and provide) npm or tr-wikipedia without turning off providing and without significantly impacting finding content 
+
+### Datastore works efficiently for large data sets
+* Final go/no-go decision on Badger (full design comparison vs other options + final decision)
```

Comment:
This should really be something like:

A fast, stable datastore capable of supporting large data sets > (?)TB has been selected and released as experimental

Reply:
That's pretty much where badger is right now

Reply:
Right - good point. Maybe this OKR should be about a new _default_ datastore that has the right characteristics for large data sets.